packages/sdk_connectivity_cubit/lib/src/sync_manager.dart

```diff
@@ -3,10 +3,12 @@ import 'dart:async';
 import 'package:connectivity_plus/connectivity_plus.dart';
 import 'package:flutter_breez_liquid/flutter_breez_liquid.dart';
 import 'package:flutter_fgbg/flutter_fgbg.dart';
+import 'package:logging/logging.dart';
 
 const syncIntervalSeconds = 60;
 
 class SyncManager {
+  final _logger = Logger("SyncManager");
```

Comment:
You're usually initialising the Logger outside the class, just checking it's intended

Reply:
Dart analyzer was causing scope issues but that's no longer the case. Moved it outside.